### PR TITLE
Weekly news

### PR DESCRIPTION
**Pull request by everett-toews**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-4a3a536195/). :bow:

File: _posts/2016-06-17-weekly-news-dockercon-and-release-information.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Weekly news: Docker 1.11.2 and DockerCon"
+date: 2016-06-17
+comments: true
+author: Everett Toews <everett.toews@rackspace.com>
+published: true
+excerpt: >
+  We released Docker Engine 1.11.2 today, which includes a key bug fix for containers that need to restart. We'll also be at DockerCon next week so if you're attending, drop by the booth to say hi and get a chance to win a Parrot Bebop Quadcopter.
```
Comment (kmsholcomb) [inline _posts/2016-06-17-weekly-news-dockercon-and-release-information.md]: ... a key fix ...
... DockerCon next week, so if ...
... Quadcopter Drone.

File: _posts/2016-06-17-weekly-news-dockercon-and-release-information.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+{{ page.excerpt }}
+
+### Docker 1.11.2 and release notes
+
+We deployed Docker Engine 1.11.2 today.
+
+![Deploy like no on is watching]({% asset_path weekly-news/friday-deploy.jpg %})
+
+Here are the [release notes](https://github.com/docker/docker/releases/tag/v1.11.2) and a [side-by-side comparison](https://github.com/docker/docker/compare/v1.11.1...v1.11.2) of the different versions in the Docker Engine repo.
```
Comment (kmsholcomb) [inline _posts/2016-06-17-weekly-news-dockercon-and-release-information.md]: For more information, see the [release notes](https://github.com/docker/docker/releases/tag/v1.11.2) and ...

File: _posts/2016-06-17-weekly-news-dockercon-and-release-information.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+It includes a key bug fix for [containers that fail to start after a daemon restart if they depend on a containerized cluster store](https://github.com/docker/docker/pull/22561). Now when you set `restart=always` your containers should always restart, even if the host reboots.
```
Comment (kmsholcomb) [inline _posts/2016-06-17-weekly-news-dockercon-and-release-information.md]: Docker Engine 1.11.2 includes a key fix for ... 
... when you set `restart=always`, your containers

File: _posts/2016-06-17-weekly-news-dockercon-and-release-information.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+To stay up-to-date with ongoing release notes, maintenance, and any incidents, see our [status page](https://status.getcarina.com/).
+
+### DockerCon 2016
+
+![Screenshot of DockerCon]({% asset_path "2016-06-03-weekly-news-pycon-and-release-information/DockerCon.png" %})
+
+We're representing at [DockerCon](http://2016.dockercon.com/) next week, so come to the expo and chat with us about Carina and all things containers. You can enter into our Carina contest, with a chance to win a Parrot Bebop Quadcopter Drone. Your neighbors will love you.
```
Comment (kmsholcomb) [inline _posts/2016-06-17-weekly-news-dockercon-and-release-information.md]: You can enter our Carina contest ...

**Comment by kmsholcomb** [comment]
Hey @everett-toews. Just a few suggested edits, in the file. Thanks!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-e7fe443e74/)

**Pull request merged**